### ISSUE TITLE
[bug] Error 405 "Method 'POST' is not supported" on the Edit Delivery page in monolithic-layered app

### ISSUE THREAD

**Issue by naXa777** [aabarmin/epam-microservices-training-2022#9]
### Steps

```sh
cd .\monolithic\monolithic-layered\
mvn spring-boot:run
```

Navigate to http://localhost:8080/delivery/1

Click "Save"

### Demo

https://github.com/aabarmin/epam-microservices-training-2022/assets/3858548/30abf5eb-0423-4bd6-8917-97915910c490

### Stacktrace

```
There was an unexpected error (type=Method Not Allowed, status=405).
Method 'POST' is not supported.
org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'POST' is not supported
	at org.springframework.web.servlet.mvc.method.RequestMappingInfoHandlerMapping.handleNoMatch(RequestMappingInfoHandlerMapping.java:265)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.lookupHandlerMethod(AbstractHandlerMethodMapping.java:441)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.getHandlerInternal(AbstractHandlerMethodMapping.java:382)
	at org.springframework.web.servlet.mvc.method.RequestMappingInfoHandlerMapping.getHandlerInternal(RequestMappingInfoHandlerMapping.java:126)
	at org.springframework.web.servlet.mvc.method.RequestMappingInfoHandlerMapping.getHandlerInternal(RequestMappingInfoHandlerMapping.java:68)
	at org.springframework.web.servlet.handler.AbstractHandlerMapping.getHandler(AbstractHandlerMapping.java:505)
...
```

### Possible solution?

Change `DeliveryController` to implement `CrudController<Delivery>` instead of `ViewAllSupport<Delivery>`

**Comment by youngInnovator** [comment]
Kindly assign this bug to me

**Comment by aabarmin** [comment]
Sure, please go ahead @youngInnovator

**Comment by youngInnovator** [comment]
Implemented in #34
